datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,395 @@
+            phone: data?.corpUser?.editableProperties?.phone || undefined,
+        };
     };
 
+    // About Text Change
+    const aboutMeTextChangeFunction = (inputString) => {
+        setEditableAboutMeText(inputString);
+        updateCorpUserPropertiesMutation({
+            variables: {
+                urn: data?.corpUser?.urn || '',
+                input: {
+                    aboutMe: inputString,
+                },
+            },
+        })
+            .catch((e) => {
+                message.destroy();
+                message.error({ content: `Failed to Save changes!: \n ${e.message || ''}`, duration: 3 });
+            })
+            .finally(() => {
+                message.success({
+                    content: `Changes saved.`,
+                    duration: 3,
+                });
+                refetch();
+            });
+    };
     return (
         <>
             {contentLoading && <Message type="loading" content="Loading..." style={messageStyle} />}
-            {data && data.corpUser && (
-                <LegacyEntityProfile
-                    title=""
-                    tags={null}
-                    header={getHeader(data.corpUser as CorpUser)}
-                    tabs={getTabs()}
+            <UserProfileWrapper>
+                <Row>
+                    <Col xl={5} lg={5} md={5} sm={24} xs={24}>
+                        <UserSidebar>
+                            <UserSidebarSubSection>
+                                <CustomAvatar
+                                    size={160}
+                                    photoUrl={profileSrc}
+                                    name={data?.corpUser?.info?.fullName || undefined}
+                                    style={{ marginTop: '14px' }}
+                                />
+                                <UserName>{data?.corpUser?.info?.fullName || 'NA'}</UserName>
+                                <UserRole>{data?.corpUser?.info?.title || 'NA'}</UserRole>
+                                <UserTeam>{data?.corpUser?.editableProperties?.teams?.join(',') || 'NA'}</UserTeam>
+                                <Divider className="divider-infoSection" />
+                                <UserSocialDetails>
+                                    <Space>
+                                        <MailOutlined /> {data?.corpUser?.info?.email || 'NA'}
+                                    </Space>
+                                </UserSocialDetails>
+                                <UserSocialDetails>
+                                    <Space>
+                                        <SlackOutlined /> {data?.corpUser?.editableProperties?.slack || 'NA'}
+                                    </Space>
+                                </UserSocialDetails>
+                                <UserSocialDetails>
+                                    <Space>
+                                        <PhoneOutlined /> {data?.corpUser?.editableProperties?.phone || 'NA'}
```

Comment:
instead of `NA`, can we show `not provided` in grey + italics?

Reply:
Sure. Text changes added. 
<img width="285" alt="Screen Shot 2022-02-15 at 2 36 43 PM" src="https://user-images.githubusercontent.com/29713365/154029196-efbdf209-0dcf-4cd8-9096-f100685418ad.png">

Reply:
Can we just do "None"

Reply:
Sure, Text change from `not provided` to `None`.